resources/js/dashboard_graph.js

```diff
@@ -25,104 +25,86 @@ $.ajax({
-          datasets: [
-          {
-              label: 'Owners',
-              data: [55, 45, 73, 75, 41, 45, 58, 73, 75, 41, 45, 58],
-              backgroundColor: "#CAC2C7"
-            }
-          ]
-        },
-        options: {
-          title: {
-            display: true,
-          },
-          scales: {
-            yAxes: [{
-              ticks: {
-                suggestedMax: 100,
-                suggestedMin: 0,
-                stepSize: 10,
-                callback: function(value, index, values){
-                  return  value
-                }
-              }
-            }]
-          },
-        }
-      });
+    const ctx = document.getElementById("ownersChart").getContext('2d');
+    const userChart = new Chart(ctx, {
+      type: 'bar',
+      data: {
+        labels: data.labels,
+        datasets: data.datasets
+        
+      },
+    });
+  },
+  error: (error)=>{
+     console.log(error);
+  }
+});
 
     //Restaurant
-      var ctx = document.getElementById("restaurantsChart");
-      var myBarChart = new Chart(ctx, {
-        type: 'bar',
-        data: {
-          labels: ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN', 'JUL', 'AUG', 'SEP', 'OCT', 'NOV', 'DEC'],
-          datasets: [
-          {
-              label: 'Restaurant pages',
-              data: [55, 45, 73, 75, 41, 45, 58, 73, 75, 41, 45, 58],
-              backgroundColor: "#CAC2C7"
-            }
-          ]
-        },
-        options: {
-          title: {
-            display: true,
-          },
-          scales: {
-            yAxes: [{
-              ticks: {
-                suggestedMax: 100,
-                suggestedMin: 0,
-                stepSize: 10,
-                callback: function(value, index, values){
-                  return  value
-                }
-              }
-            }]
+    $.ajax({
+      headers: {
+        'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+      },
+      url:'/api/admin/dashboard_restaurant',
+      type:'GET',
+      data: {
+        user_type: 'restaurant'
+      },
+      success: (data)=>{
+    
+        const ctx = document.getElementById("restaurantChart").getContext('2d');
+        const userChart = new Chart(ctx, {
+          type: 'bar',
+          data: {
+            labels: data.labels,
+            datasets: data.datasets
+            
           },
-        }
-      });
+        });
+      },
+      error: (error)=>{
+         console.log(error);
+      }
+    });
 
   // Reviews
-  var ctx = document.getElementById("reviewsChart");
-  var myBarChart = new Chart(ctx, {
-    type: 'bar',
-    data: {
-      labels: ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN', 'JUL', 'AUG', 'SEP', 'OCT', 'NOV', 'DEC'],
-      datasets: [
-      {
-          label: 'Number of reviews',
-          data: [55, 45, 73, 75, 41, 45, 58, 73, 75, 41, 45, 58],
-          backgroundColor: "#CAC2C7"
-        }
-      ]
-    },
-    options: {
-      title: {
-        display: true,
+
```

Comment:
Should this section be a reviews?
Because we already have the ```user_type: 'restaurant'``` at the top. The below code is redundant. Perhaps is it for the reviews?